share/doc/wake/basic_wake_tutorial.md

```diff
@@ -0,0 +1,809 @@
+## Wake Tutorial
+
```

Comment:
Can you add an intro here, like "This tutorial will teach you ...... After completing this tutorial, you will have .....

Right now it is something like "This tutorial will teach you the basics of the wake language. ... you will have typed and executed many basic wake functions".

Reply:
Sure. I will add this.

Reply:
I will add this